src/pke/examples/polynomial-evaluation.py

```diff
@@ -0,0 +1,70 @@
+from openfhe import *
+import time
+
+def main():
+
+    print("\n======EXAMPLE FOR EVALPOLY========\n")
+    parameters = CCParamsCKKSRNS()
+    parameters.SetMultiplicativeDepth(6)
+    parameters.SetScalingModSize(50)
+
+    cc = GenCryptoContext(parameters)
+    cc.Enable(PKESchemeFeature.PKE)
+    cc.Enable(PKESchemeFeature.KEYSWITCH)
+    cc.Enable(PKESchemeFeature.LEVELEDSHE)
+    cc.Enable(PKESchemeFeature.ADVANCEDSHE)
+
+    input = [complex(a,0) for a in [0.5, 0.7, 0.9, 0.95, 0.93]]
```

Comment:
@reneroliveira 

Would it be difficult to make the code also accept doubles and integers? 

**Doubles**: On the C++ side, you can pass in a vector of doubles and OpenFHE will figure out how to convert things. 

**Integers**: python's dynamic typing means that an end-user probably doesn't want to have to care about integers vs. floats vs. complex. I think it would be more pythonic if we were to integrate the changes I mentioned.